tests/integration/buildcmd/build_integ_base.py

```diff
@@ -172,6 +172,8 @@ def _verify_invoke_built_function(self, template_path, function_logical_id, over
         process_execute.process.wait()
 
         process_stdout = process_execute.stdout.decode("utf-8")
+        if process_stdout.startswith("Config file"):
+            *_, process_stdout = process_stdout.partition("\n")
```

Comment:
Do we need "if" here since "Config file" is always printed?

Reply:
Well it's not exactly every time.  There are commands which don't use it so it's better be on the safe side